src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -3248,6 +3439,14 @@ enum AbsoluteDirection {
   NORTHWEST
 }
 
+input ScooterPreferences {
```

Comment:
Are you putting that in just for good measure? I don't think we have that implemented at the moment, isn't it?

Reply:
I added this for two reasons: so it's more clear what settings should be defined when you want to adjust scooter travel and so we can in the future support defining different parameters for cycling/scooter. However, on the downside, what should happen now if both cycling and scooter settings are included.